server/channels/api4/channel.go

```diff
@@ -843,6 +846,18 @@ func getAllChannels(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func SanitizeChannels(c *Context, channels model.ChannelListWithTeamData) model.ChannelListWithTeamData {
```

Comment:
0/5 but we might want to rename this to something like `sanitizeAllChannelsResponse` to be extra clear that we aren't intending it to sanitize every list of channels returned by the API

Reply:
This is a great point, @sbishel I would also join this recommendation

Reply:
Done